### PR TITLE
Added remaining content with milestones and dependencies

### PR DESCRIPTION
**Pull request by bentley-ritson-umuzi**

### CODE REVIEW

File: content/topics/kotlin/classes/_index.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
+---
+title: Classes 
+ready: true
+prerequisites:
+  hard: ["topics/kotlin/basic-control-flow"]
+  soft: []
+
+---
+## Classes
```
Comment (sheenarbw) [inline content/topics/kotlin/classes/_index.md:9]: I'm not 100% convinced that you looked at how this content would render. The html would start off looking like this:

```
<h1>Classes</h1>
<h2>Classes</h2>
```
This is redundat redundant

File: content/topics/kotlin/classes/_index.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+A class in Kotlin can have a primary constructor and one or more secondary constructors.
+
+
+````
+class Person constructor(firstName: String) {
+    }
+
+````
+
+Secondary Constructors
```
Comment (sheenarbw) [inline content/topics/kotlin/classes/_index.md:27]: Also this. I think the html would look like:
```
<p> Secondary Constructors The class can also declare secondary constructors, which are prefixed with constructor:
</p>
```

There are a few other places where this is the case.

Better would be:

```
### Secondary Constructors
etc
```

File: content/topics/kotlin/interface/_index.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+An interface is defined using the keyword interface
+````
+interface MyInterface {
+    fun bar()
+    fun foo() {
+    // optional body
+    }
+    }
+````
+Implementing Interfaces
```
Comment (sheenarbw) [inline content/topics/kotlin/interface/_index.md:20]: also here

File: content/topics/kotlin/interface/_index.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
+---
+title: Interface 
+ready: true
+prerequisites:
+  hard: ["topics/kotlin/properties and fields"]
```
Comment (sheenarbw) [inline content/topics/kotlin/interface/_index.md:5]: Naming conventions.

  hard: ["topics/kotlin/properties-and-fields"]

File: content/topics/kotlin/classes/_index.md (hunk 1 above)
Comment (bentley-ritson-umuzi) [reply content/topics/kotlin/classes/_index.md:9]: Oh I totally forgot about that little detail. I did check just dint notice.

File: content/topics/kotlin/classes/_index.md (hunk 2 above)
Comment (bentley-ritson-umuzi) [reply content/topics/kotlin/classes/_index.md:27]: Yea I missed a few was typing really fast.

File: content/topics/kotlin/basic-control-flow/_index.md [hunk 5]
```diff
@@ -1,14 +1,12 @@
...
   hard: ["topics/kotlin/basic-syntax-types"]
   soft: []
 ---
 
-**BASICS - Control Flow**
-
-If Expression
+### If Expression
 
 In Kotlin, if is an expression, i.e. it returns a value. Therefore there is no ternary operator (condition ? then : else), because ordinary if works fine in this role.
```
[the remark this answers is not in the document]
Comment (bentley-ritson-umuzi) [reply content/topics/kotlin/basic-control-flow/_index.md]: noted and implemented.

**Pull request merged**